source-code/sdk-js/src/config/index.ts

```diff
@@ -0,0 +1,42 @@
+import { DefineConfig, initialize$import } from "@inlang/core/config"
+import fs from "node:fs/promises"
+import { resolve } from "node:path"
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => undefined
```

Comment:
what is noop for?

Reply:
I will inline it. Previously I have used it in more places.